templates/named.conf.erb

```diff
@@ -181,3 +181,8 @@ include "<%= filename %>";
 <% end -%>
 <% end -%>
 
+<% if !@statistics_channels.empty? -%>
```

Comment:
I think this check might fail with `undefined method #empty? for NilClass` or something similar like that. You should be able to just `<% if @statistics_channels %>`